platform-cloud/docs/compute-envs/seqera-compute.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Seqera Compute"
+description: "Instructions to set up Seqera Compute in Seqera Platform"
+date: "21 Apr 2025"
+tags: [seqera compute, compute environment]
+---
+
+Seqera Compute enables you to run pipelines and studio sessions in Seqera Cloud in a fully managed and optimized AWS environment. Seqera automatically provisions and manages all the underlying resources, including AWS accounts, credentials, roles, AWS Batch compute environments, and S3 storage buckets, requiring minimal user configuration. Using prepaid credits enables control and visibility of the compute spend in each of your organization's workspaces.
```

Comment:
Note – Can we avoid calling out 'Batch' specifically as this won't always be the case? We can just say "AWS compute environments"